ChangeLog

```diff
@@ -1,3 +1,41 @@
+# 2025-02-06 Version 3.11.0
+
+A new release with bugfixes and code cleanups as well as a few nifty little
+features that will improve your meetings.
+
+Noteworthy changes:
+ * Updated android client to more recent gradle (#11105, #11110)
+ * Fix cmake clean target (#109
+ * SDL3 bugfixes and API updates (#11092, #11093, #11128)
+ * Fix keyboard mapping, add working japanese and korean types, deprecate
+   obsolete functions (#10989, #11035, #11011, #11074, #11037)
+ * Fix timezone mapping and iteration (#11077, #11079, #11080, #11083)
+ * Fix YUV filter for AVC444 modes (#11045, #11063, #11066, #11081, #11086,
```

Comment:
Nitpick:
```suggestion
 * Fix YUV reverse filter for AVC444 modes (#11045, #11063, #11066, #11081, #11086,
```

(people, that don't develop in the RDP space (99.9%) won't notice though :P)